src/assets/scripts/templates/product.js

```diff
@@ -1 +1,8 @@
-// import '../sections/product';
+import '../sections/product';
+
+import $ from 'jquery';
+import sections from '@shopify/theme-sections';
+
+$(document).ready(() => {
+  sections.load('*');
```

Comment:
Not sure but we can probably pass just the product section, i.e. ` sections.load('product');`, instead of loading all sections?

Reply:
You're right!